tests/migration_test.go

```diff
@@ -103,6 +103,22 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		if len(nodes.Items) < 2 {
 			Skip("Migration tests require at least 2 nodes")
 		}
+
+		// Taints defined by k8s are special and can't be applied manually.
+		// Temporarily configure KubeVirt to use something else for the duration of these tests.
+		if tests.IsUsingBuiltinNodeDrainKey() {
```

Comment:
Do we need to clean this after the tests?

Reply:
Nope! there's a general "revert any changes to the configmap" step in the AfterEach already.

Reply:
Ah, that's right. Great.
/lgtm